src/System.Security.Cryptography.X509Certificates/tests/CertTests.cs

```diff
@@ -80,29 +88,65 @@ public static void X509Cert2Test()
         }
 
 #if netstandard17
-        [Fact]
-        public static void TestVerify()
+        [OuterLoop( /* May require using the network, to download CRLs and intermediates */)]
```

Comment:
I'm pretty sure that you still need [Fact] with [OuterLoop].

And the new style (which didn't exist when I was doing the C-style comment in parens) is to use a literal string there.